vss-extension.json

```diff
@@ -61,7 +61,13 @@
       "uri": "https://github.com/infracost/infracost-azure-devops/issues"
     },
     "support": {
-      "uri": "mailto:hello@infracost.io"
+      "uri": "https://github.com/infracost/infracost-azure-devops/issues"
```

Comment:
It might be better to link to https://www.infracost.io/docs/support/ as that'll include links to the troubleshooting guide too